hubblestack/extmods/modules/hubble.py

```diff
@@ -671,13 +671,14 @@ def load():
     log.debug('loading nova modules')
 
     global __nova__
-    __nova__ = NovaLazyLoader(_hubble_dir(), __opts__, __grains__, __pillar__, __salt__)
+    __nova__ = NovaLazyLoader(_hubble_dir(), __opts__, __mods__)
 
     ret = {'loaded': list(__nova__._dict.keys()),
```

Comment:
Still typo: "_dict"

Reply:
That's actually not a typo. I thought it was when I commented on it, but it's actually something that's populated in salt.utils.lazy in LazyDict